src/framework/shortcuts/qml/MuseScore/Shortcuts/EditShortcutDialog.qml

```diff
@@ -141,6 +151,8 @@ StyledDialogView {
 
                 buttons: [ ButtonBoxModel.Cancel, ButtonBoxModel.Save ]
 
+                navigationPanel.section: root.navigationSection
```

Comment:
also you should set order for panel - 2

Reply:
setting order here disturbs navigation

Reply:
I meant navigationPanel.order: 2

Reply:
@Eism Understood but Setting the order here results in the following behavior:
- When Dialog opened, it focuses on the bottom panel, i.e. save button.
- On pressing tab, it navigates to the text input field.

But we want it to first focus on the text input field and then on the bottom panel, right?

Reply:
Checked - works fine for me

To clarify, did you add `navigationPanel.order: 2` after this line and it's not working for you?

Reply:
@Eism Yes Rechecked,  by adding this order results into above behavior, Is this Acceptable ?

Reply:
let's wait for what @DmitryArefiev says